gui/window_character.gui

```diff
@@ -726,7 +726,9 @@ window = {
 								tooltip = "[Character.GetUINameNotMeNoTooltip]"
```

Comment:
You can find out the age in the character finder.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/114200539-5c594400-9966-11eb-8b63-b49a0f1ae9cc.png)

</details>